.eslintrc.json

```diff
@@ -8,6 +11,7 @@
         "ignoreConsecutiveComments": true
       }
     ],
-    "no-console": "warn"
+    "no-console": "warn",
+    "react/no-danger": "off"
```

Comment:
@maxpluez Rather than turn it off globally, can you just ignore the rule in the particular part of the code that violates this? We don't want people to violate this without knowingly doing so